moto/route53/models.py

```diff
@@ -264,20 +270,20 @@ def delete_rrset_by_id(self, set_identifier):
         ]
 
     def get_record_sets(self, start_type, start_name):
-        record_sets = list(self.rrsets)  # Copy the list
+        def predicate(rrset):
+            return (
+                reverse_domain_name(rrset.name) < reverse_domain_name(start_name)
+                or rrset.type_ < start_type
```

Comment:
This should be an _and_-condition

Reply:
Because this predicate is used in `dropwhile` function it will drop record sets until the first one which has name >= start_name and type >= start_type will be found. So the condition is correct.

Reply:
Gotcha, my bad